documentation/asciidoc/accessories/iqaudio/advanced.adoc

```diff
@@ -1,36 +1,35 @@
 == Hardware information
 
-=== GPIO usage / Pin headers
+=== GPIO usage 
 
-Raspberry Pi Audio boards take advantage of a number of Raspberry Pi Pi signals in
-order to operate successfully. Some of these signals are solely for the use of the board and
+Raspberry Pi audio boards take advantage of a number of pins on the GPIO expander in
+order to operate successfully. Some of these pins are solely for the use of the board, and
 some can be shared with other peripherals, sensors, etc.
 
 The following Raspberry Pi GPIO pins will be used by the audio boards:
 
-* ALL POWER PINS
-* ALL GROUND PINS
+* All power pins
+* All ground pins
 * GPIO 2/3 (I2C)
 * GPIO 18/19/20/21 (I2S)
```

Comment:
Presumably the EEPROM-ID pins are also used? GPIO 0 and 1.

Reply:
That'll need some input from one of the engineers - leaving for now.